src/errors.rs

```diff
@@ -75,7 +75,7 @@ pub enum BinstallError {
     /// - Exit: 74
     #[error(transparent)]
     #[diagnostic(severity(error), code(binstall::io))]
-    Io(#[from] std::io::Error),
+    Io(std::io::Error),
```

Comment:
use `#[source]` here even if no `#[from]`

Reply:
Using source here gives compilation error.

I think error(transparent) is enough here.

Reply:
Oh, you're right.